### ISSUE TITLE
Cannot force cabal sandbox hc-pkg unregister

### ISSUE THREAD

**Issue by bitemyapp** [haskell/cabal#2379]
```
[callen@atlantis ~/Work/frontrowed/fancy-api]$ cabal sandbox hc-pkg unregister frontrow-entities
ghc-pkg: unregistering frontrow-entities would break the following packages: fancy-api-0.0.1 (use --force to override)
[callen@atlantis ~/Work/frontrowed/fancy-api]$ cabal sandbox hc-pkg unregister --force frontrow-entities
cabal: unrecognized 'sandbox' option `--force'
[callen@atlantis ~/Work/frontrowed/fancy-api]$ cabal sandbox hc-pkg --force unregister frontrow-entities 
cabal: unrecognized 'sandbox' option `--force'
[callen@atlantis ~/Work/frontrowed/fancy-api]$ cabal sandbox hc-pkg unregister frontrow-entities --force 
cabal: unrecognized 'sandbox' option `--force'
```

Cabal is refusing to rebuild my add-source dependency (Cf. https://github.com/haskell/cabal/issues/2378)

**Comment by bitemyapp** [comment]
cabal sandbox hc-pkg -- --force unregister frontrow-entities

This should be added to the help for sandbox.

**Comment by bitemyapp** [comment]
help has:

```
 cabal sandbox hc-pkg        [FLAGS] [--] COMMAND [--] [ARGS]
```

An example would be more noticeable.

**Comment by lspitzner** [comment]
fixed (as part of #2416): `cabal help sandbox` now lists one example using `--force` 

~~~~
> cabal help sandbox
[..]
Examples:
  Set up a sandbox with one local dependency, located at ../foo:
    cabal sandbox init
    cabal sandbox add-source ../foo
    cabal install --only-dependencies
  Reset the sandbox:
    cabal sandbox delete
    cabal sandbox init
    cabal install --only-dependencies
  List the packages in the sandbox:
    cabal sandbox hc-pkg list
  Unregister the `broken` package from the sandbox:
    cabal sandbox hc-pkg -- --force unregister broken
~~~~
